doc/performance-testing/cloud-run/0001_stress-test-december-2021.md

```diff
@@ -0,0 +1,86 @@
+# Stress Test December 2021
+This is an initial stress test to asses the performance of runner after the switch from kubernetes to cloud run. Below are the initial settings we used:
+
+
+## Runner settings
+
+| Setting | Value |
+| --- | ---| 
+| Concurrency | 250 |
+| Max instances   | 1 |
+| Min instances | 1 |
+
+
+## Benchmark settings
+
+| Setting | Value |
+| --- | ---| 
+| Load injector instances | 1 |
+| Clients hatch rate   | 50 |
+| Wait time minimum | 1 |
+| Wait time maximum | 2 |
+| Requests JSON | test_benchmark_business.json |
+| Runtime | 10m |
```

Comment:
The other PR it was 20 minutes, probably worth making the default 10 minutes on that one to avoid confusion

Reply:
Changed it to 10 minutes in the other PR, thanks!